css-typed-om/Overview.bs

```diff
@@ -351,6 +351,29 @@ Issue: There are now 3 ways of creating Length values -
 
 Issue: What happens if the provided DOMString or LengthValue for (e.g.) a SimpleLength is invalid?
 
+{{ColorValue}} objects {#colorvalue-objects}
+---------------------------------------------
+<pre class='idl'>
+
+[Constructor(double r, double g, double b),
+ Constructor(double r, double g, double b, double a)]
```

Comment:
I wonder whether we should allow constructing `ColorValue` with supported color keywords. Or is there any existing function to do so?